Create CODE_OF_CONDUCT.md (#160)
aep-dev/aep.dev: CODE_OF_CONDUCT.md

- [French/Français](code-of-conduct-languages/fr.md)
- [German/Deutsch](code-of-conduct-languages/de.md)
- [Hindi/हिन्दी](code-of-conduct-languages/hi.md)
- [Indonesian/Bahasa Indonesia](code-of-conduct-languages/id.md)
- [Italian/Italiano](code-of-conduct-languages/it.md)
- [Japanese/日本語](code-of-conduct-languages/ja.md)
- [Korean/한국어](code-of-conduct-languages/ko.md)
- [Polish/Polski](code-of-conduct-languages/pl.md)
- [Portuguese/Português](code-of-conduct-languages/pt.md)
- [Russian/Русский](code-of-conduct-languages/ru.md)
- [Spanish/Español](code-of-conduct-languages/es.md)
- [Turkish/Türkçe](code-of-conduct-languages/tr.md)
- [Ukrainian/Українська](code-of-conduct-languages/uk.md)
- [Vietnamese/Tiếng Việt](code-of-conduct-languages/vi.md)

### Community Code of Conduct

As contributors, maintainers, and participants in the CNCF community, and in the interest of fostering
an open and welcoming community, we pledge to respect all people who participate or contribute
through reporting issues, posting feature requests, updating documentation,
submitting pull requests or patches, attending conferences or events, or engaging in other community or project activities.

> We are committed to making participation in the CNCF community a harassment-free experience for everyone, regardless of age, body size, caste, disability, ethnicity, level of experience, family status, gender, gender identity and expression, marital status, military or veteran status, nationality, personal appearance, race, religion, sexual orientation, socioeconomic status, tribe, or any other dimension of diversity.

Reviewer: ```suggestion
We are committed to making participation in the CNCF community a harassment-free experience for everyone, regardless of individual characteristics.
```

Seems more maintainable than listing every possible personal characteristic upon which someone could discriminate?

Reviewer 2: I don't think we should modify the CNCF COC - if we do join the CNCF we'd have to revert this modification anyway, and if we end up with a separate organization we may have to adopt theirs.

Also in practice I think most people will look at this file, see it's basically the CNCF COC, and not read through the whole thing. So I'm not sure if there's value to someone to modify this file?

Reviewer: If the goal is to adopt the CNCF CoC, rather than to write an AEP and use the CNCF one as a starting point, then I agree we shouldn't modify it.

## Scope

This code of conduct applies:
* within project and community spaces,

> * in other spaces when an individual CNCF community participant's words or actions are directed at or are about a CNCF project, the CNCF community, or another CNCF community participant.

Reviewer: Is this applicable?

### CNCF Events

> CNCF events that are produced by the Linux Foundation with professional events staff are governed by the Linux Foundation [Events Code of Conduct](https://events.linuxfoundation.org/code-of-conduct/) available on the event page. This is designed to be used in conjunction with the CNCF Code of Conduct.

Reviewer: Is this applicable?

- Unwelcome physical contact
- Unwelcome sexual or romantic attention or advances
- Other conduct which could reasonably be considered inappropriate in a
  professional setting

The following behaviors are also prohibited:

- Providing knowingly false or misleading information in connection with a Code
  of Conduct investigation or otherwise intentionally tampering with an
  investigation.
- Retaliating against a person because they reported an incident or provided
  information about an incident as a witness.

Project maintainers have the right and responsibility to remove, edit, or
reject comments, commits, code, wiki edits, issues, and other contributions
that are not aligned to this Code of Conduct. By adopting this Code of Conduct,
project maintainers commit themselves to fairly and consistently applying these
principles to every aspect of managing a project. Project maintainers who do
not follow or enforce the Code of Conduct may be temporarily or permanently
removed from the project team.

## Reporting

For incidents occurring in our community, please contact any of the individual
maintainers of the GitHub organization or Slack workspace.

## Enforcement

> Upon review and investigation of a reported incident, the CoC response team

Reviewer 2: nit: we don't have a CoC respone team to do we? maybe just make this "AEP maintainers".

> Upon review and investigation of a reported incident, the CoC response team

Author: ```suggestion
Upon review and investigation of a reported incident, the AEP maintainers
```
